### PR TITLE
mlt: 6.10.0 -> 6.12.0

### PR DESCRIPTION
**Pull request by dtzWill**
###### Motivation for this change

https://github.com/mltframework/mlt/releases/tag/v6.12.0

Notes mention new VA-API support, and other goodies..
among many important fixes! :)

Should look through them more closely,
possibly adding new deps (for vaapi?).

(note to myself but open to anyone with a second :))

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/libraries/mlt/default.nix [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
 
   src = fetchFromGitHub {
     owner = "mltframework";
     repo = "mlt";
     rev = "v${version}";
-    sha256 = "0ki86yslr5ywa6sz8pjrgd9a4rn2rr4mss2zkmqi7pq8prgsm1fr";
+    sha256 = "0pzm3mjbbdl2rkbswgyfkx552xlxh2qrwzsi2a4dicfr92rfgq6w";
   };
 
   buildInputs = [
```
Comment (c0bw3b) [inline pkgs/development/libraries/mlt/default.nix]: At least `makeWrapper` and `pkgconfig` should be into `nativeBuildInputs`

**Comment by c0bw3b** [comment]
The configure flag `--enable-gpl` is repeated twice?

**Comment by dtzWill** [comment]
> The configure flag `--enable-gpl` is repeated twice?

That's how you enable gpl x2 => gplv2 code, right?! :rofl:

Cleaning up, thank you for review and apologies for missing these things!
Oh hooray: in my defense the configureArgs were already dup'd haha.
Okay I'll check they don't somehow actually help xD.

**Comment by dtzWill** [comment]
Seems we half-upgraded this by bumping qt-enabled variant in b0f912b6d94aa56ada9065a2676f6bd71a7cfecb or so.  Rebasing/fixing underway...

**Comment by dtzWill** [comment]
Merged the two mlt expressions, which seems high-level good (as well as a few dependencies that seem no heavier than what's already pulled in) but should make sure shotcut and kdenlive both still work.  I nuked the "fix motiv.dev" postInstall as likely kruft but will revisit.

**Comment by dtzWill** [comment]
This seems to have suffered some bitrot, if we have a better PR or so feel free to close :).  Hopefully will be revisited eventually if not :).

**Comment by SuperSandro2000** [comment]
Master already has version 6.22.1. If you want to move any of the other changes over please open a new PR.

**Pull request closed without merging**